src/style.css

```diff
@@ -69,3 +69,20 @@ h2 {
   font-weight: 400;
   font-size: 1.8rem;
 }
+
+.form-wrapper {
+  position: fixed;
+  left: 80rem;
+  top: 10rem;
+  width: 40rem;
+}
+
+.form-position {
+  position: fixed;
+  right: 107rem;
+  top: 13rem;
+}
```

Comment:
- [ OPTIONAL] Although the styling is not required for this milestone, the CSS declarations for this selector have made it difficult to see all the scores I posted to the endpoint when I retrieve it. I would suggest that you use a more responsive layout like `flexbox` or `grid` to align the children elements of this element. 👍